### ISSUE TITLE
v10 bindings

### ISSUE THREAD

**Issue by grsoares21** [diogomqbm/rescript-react-flow#6]
Recently React Flow has released it's v10 and it contains lots of breaking changes on their API (https://reactflow.dev/docs/guides/migrate-to-v10/). Therefore we'd need new bindings to make it work with ReScript.

I'd definitely offer to make a PR for this but I'm still a beginner in ReScript and I'm not even sure how to begin. If this hasn't been tackled yet by the time I have enough knowledge to do so, I'll do it. Also I'm not really sure how the versioning works for these cases? Will it just be a new major NPM version for the bindings?

Thanks in advance if anyone decides to tackle this. I'll be using a previous version of React Flow meanwhile.

**Comment by diogomqbm** [comment]
@grsoares21 Right. So I think we should be able to deprecate the older version and launch new bindings pointing to v10. Feel free to open a PR, ask any question here or on Discord regarding this.
